database/seeder.js

```diff
@@ -5,7 +5,7 @@ const Schema = mongoose.Schema;
 
 let database = {availability: {}};
 const months = Array.from(Array(12), (el, ind) => ind);
-const years = ['2019', '2020', '2021'];
```

Comment:
Is there a more dynamic way to handle months of different lengths? Could the same thing be achieved with a loop instead of hard-coding incremented values/

Reply:
I'm not sure which line you're referring to. Line 7 generates an array [0, 1, 2, ... 11]. Lines 29 to 43 handle the number of days each month gets. I can't think of anyway to dynamically give each month its specific number of days.